public/realms/mainnet-beta.json

```diff
@@ -2941,5 +2941,14 @@
     "website": "https://www.digitaleyes.market/",
     "twitter": "@DigitalEyesNFT",
     "discord": "https://discord.com/invite/hhbgab4bsD"
+  },
+    {
+    "symbol": "HackerZ DAO",
+    "displayName": "HackerZ",
+    "programId": "GovER5Lthms3bLBqWub97yVrMmEogzX7xNjdXpPPCVZw",
+    "realmId": "A8qAQW8SVUPB51znW8Ym81AYb2TFPLCzUwgzWvpca7be",
+    "twitter": "@HackerzDAO",
+    "discord": "https://discord.gg/gK2PSJYe",
+	"ogImage": "/realms/HackerzDAO_PFP_REALMS.png"
```

Comment:
nit: formatting

```suggestion
 {
    "symbol": "HackerZ DAO",
    "displayName": "HackerZ",
    "programId": "GovER5Lthms3bLBqWub97yVrMmEogzX7xNjdXpPPCVZw",
    "realmId": "A8qAQW8SVUPB51znW8Ym81AYb2TFPLCzUwgzWvpca7be",
    "twitter": "@HackerzDAO",
    "discord": "https://discord.gg/gK2PSJYe",
    "ogImage": "/realms/HackerzDAO_PFP_REALMS.png"
```

Reply:
Thank you for the support :) Appreciated.